spec/features/user_adds_a_review_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+feature "adding review" do
+  scenario "users add a new review" do
+    user = FactoryGirl.create(:user)
+    spot = FactoryGirl.create(:spot)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+
+    visit spot_path(spot)
```

Comment:
Sorry for not noticing this before, but lines 5-14 should be extracted into a `before(:each)` action so they're not duplicated in each test